src/module.ts

```diff
@@ -50,20 +55,43 @@ export default defineNuxtModule<ModuleOptions>({
       'useUser'
     ].map(key => ({
       name: key,
-      from: '@vue-clerk'
+      from: 'vue-clerk'
     })))
 
+    if (installServerMiddleware) {
+      addServerHandler({
+        middleware: true,
+        handler: resolve('./runtime/middleware')
+      })
+    }
```

Comment:
This middleware is required for clerk to inject cookies and stuff. It's not recommended if SSR is set to false